.github/workflows/buildcache.yml

```diff
@@ -1,3 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to you under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
It is not directly related to this issue
However out of curiosity: should we change this url to `https` rather than having `http` ?
like https://www.apache.org/licenses/LICENSE-2.0

Reply:
We should follow https://www.apache.org/legal/src-headers.html#headers so not at this point. I recall some discussions about updating HTTP to HTTPS in the header but I don't remember in which mailing list that was.